### DISCUSSION TITLE
@types/gulp-eslint

### DISCUSSION THREAD

**Discussion by Dulux-Oz** [DefinitelyTyped/DefinitelyTyped#58307] [Request a new @types package] [answered]
Hi All,

Could someone help a (TypeScript) noob (again :-) ) with some advice, please, by answering one (or more) of the following questions:

1. Does anyone have the TypeScript def file for gulp-eslint, and may I please have a copy?
2. Could someone please write the TypeScript def file for gulp-eslint?
3. Could someone help me to write the TypeScript def file for gulp-eslint?
4. Could someone point me towards how to use gulp-eslint"as it is" with TypeScript?

Thanks in advance (& thanks again for the previous help I've been given - I really do appreciate it),

Dulux-Oz

**Comment by martin-badin** [comment, marked as answer]
I created PR #58338

**Comment by Dulux-Oz** [comment]
OK, so I've now got to go and eat some humble-pie.

I am now unsure if this is even needed now, as I've discovered a new npm package `gulp-eslint-new` which is more up-to-date than the old `gulp-eslint` and which also includes its own TS defs.

Thank you to @martin-badin and @fasttime for looking at this, but I feel that the effort may have been... unnecessary?

Sorry for being a PITA - especially as @martin-badin has been so helpful over the last couple of weeks - I feel really bad about putting you thru all the effort - sorry about that.

I'll make this as answered/done/closed and see about making an entry in the Gulp-Blacklist about using the new `gulp-eslint-new` instead of `gulp-eslint`.

Cheers

Dulux-Oz
